WordPress/Classes/Services/ReaderTopicService+Interests.swift

```diff
@@ -1,6 +1,19 @@
 import Foundation
 import WordPressKit
 
+// MARK: - ReaderInterestsService
+
+/// Protocol representing a service that retrieves a list of interests stored remotely
+protocol ReaderInterestsService: AnyObject {
+    /// Fetches a large list of interests from the server
+    /// - Parameters:
+    ///   - success: Called upon successful completion and parsing, provides an array of `RemoteReaderInterest` objects
+    ///   - failure: Called upon network failure, or parsing errors, provides an Error object
+    func fetchInterests(success: @escaping ([RemoteReaderInterest]) -> Void,
+                        failure: @escaping (Error) -> Void)
```

Comment:
The name of this method and the parameters are pretty straightforward. Do you think we really need the comment?

Reply:
Done 263b50fa21ab548902d78b95a44f46f3559121c5